installing.md

```diff
@@ -0,0 +1,288 @@
+# Installing Pelias
+
+Mapzen offers the Mapzen Search service in hopes that as many people as possible will use it,
+but we also encourage people to set up their own Pelias instance. Whether it's to import their own data,
+make their own tweaks to Pelias code, or to help with Pelias development, its important that we
+document how this can be done. Similarly, while there are ways this process can be
+[automated](https://github.com/pelias/vagrant), these instructions are written as if the setup is
+manual, to illustrate all the moving pieces of Pelias.
+
+## Gather the Ingredients
+
+In general, Pelias will require:
+
+* A working [Elasticsearch](https://www.elastic.co/products/elasticsearch) 1.7 cluster. It can be on
+  a single machine or across several
+* [Node.js](https://nodejs.org/) 0.12 or newer (Node 4 or 5 is recommended)
+* Up to 100GB disk space to download and extract data
+* Lots of RAM. At least 2-4GB. A full North America OSM import just barely fits on a machine with 16GB RAM
+
+## Choose your branch
+
+As part of the setup instructions below, you'll be downloading several Pelias packages from source
+on Github. All of these packages offer 3 branches for various use cases. Based on your needs, you
+should pick one of these branches and use the same one across all of the Pelias packages.
+
+`production`: contains only code that has been tested against a full-planet build and is live on
+Mapzen Search. This is the "safest" branch and it will change the least frequently, although we
+generally release new code at least once a week.
+
+`staging`: these branches contain the code that is currently being tested against a full planet
+build for imminent release to Mapzen Search. It's useful to track what code will be going out in the
+next release, but not much else.
+
+`master`: master branches contain the latest code that has passed code review, unit/integration
+tests, and is ready to be included in the next release. While we try to avoid it, the nature of the
+master branch is that it will sometimes be broken. That said, these are the branches to use for
+development of new features.
```

Comment:
Yes, for development. Having pull requests come in against the production branch could lead to problems.

Migurski gave similar feedback that it was not necessary to explain the different branches, but it seems like that really isn't the case. This doc is definitely meant to be read by people who would contribute back to Pelias, and explaining the different branches seems helpful either way. Does that seem reasonable?